test/e2e/framework/pod/utils.go

```diff
@@ -241,3 +241,23 @@ func FindPodConditionByType(podStatus *v1.PodStatus, conditionType v1.PodConditi
 	}
 	return nil
 }
+
+// FindContainerStatusInPod finds a container status by its name in the provided pod
+func FindContainerStatusInPod(pod *v1.Pod, containerName string) *v1.ContainerStatus {
+	for _, container := range pod.Status.InitContainerStatuses {
```

Comment:
naming here looks a bit confusing to me. Would it be better to rename container -> containerStatus?

Reply:
@mimowo any comments on this?

Reply:
Thanks for the reminder, I parked the test for a while due to a failing pull configuration and returned to it only recently. I like the suggestion, pushed the change.